api_types.ts

```diff
@@ -912,10 +912,13 @@ export type OptionsReference = string & {
   __brand: 'OptionsRef';
 };
 
-// TODO(dweitzman): Update the type to allow options functions to return a cache TTL.
+type PropertyOptionsMetadataResult<ResultT extends PackFormulaResult[]> =
+  | ResultT
+  | {results: ResultT; cacheTtlSecs?: number};
```

Comment:
For sync table `execute` and `executeUpdate` the key holding the results is `result` (singular). Perhaps use the same here?

Reply:
Consistency makes sense to me. Renamed here and updated the normalization logic. 

To avoid a breaking change, the hidden internal type `PropertyOptionsAnnotatedResult` still uses `results` (plural) but pack makers won't ever need that type.